Translations/bg.po

```diff
@@ -7132,206 +7347,209 @@ msgid ""
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:409
 msgid "Cain isn't what he says he is. Sure, sure, he talks a good story... some of 'em are real scary or funny... but I think he knows more than he knows he knows."
-msgstr ""
+msgstr "Кайн не е този, който твърди, че е. Добър разказвач е, безспорно... някои от историите са потресаващи, други смешни....  Мен ако питаш, знае много повече от това, което твърди, че знае."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:411
 msgid "Griswold? Good old Griswold. I love him like a brother! We fought together, you know, back when... we... Lazarus...  Lazarus... Lazarus!!!"
-msgstr ""
+msgstr "Грисволд? Добрият стар Грисволд. Обичам го като свой брат! Бихме се рамо до рамо навремето, когато.... двамата.... Лазар... Лазар... Лазар!!!"
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:413
 msgid "Hehehe, I like Pepin. He really tries, you know. Listen here, you should make sure you get to know him. Good fella like that with people always wantin' help. Hey, I guess that would be kinda like you, huh hero? I was a hero too..."
-msgstr ""
+msgstr "Ха-ха-ха, харесвам Пипин. Знаеш ли, поне наистина се старае. Да знаеш, трябва да го опознаеш по-отблизо. Добър човечец, който винаги е при онези, които се нуждаят от помощ. Ей, мисля си, че е като теб, герой, а? И аз бях герой..."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:415
 msgid "Wirt is a kid with more problems than even me, and I know all about problems. Listen here - that kid is gotta sweet deal, but he's been there, you know? Lost a leg! Gotta walk around on a piece of wood. So sad, so sad..."
-msgstr ""
+msgstr "Върт си има повече проблеми, дори и от мен, знам за всички тях. Послушай ме, това хлапе - той беше там, нали разбираш? Загуби си крака! Сега ходи на дървен кол. Колко жалко, колко жалко..."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:417
 msgid "Ogden is the best man in town. I don't think his wife likes me much, but as long as she keeps tappin' kegs, I'll like her just fine. Seems like I been spendin' more time with Ogden than most, but he's so good to me..."
-msgstr ""
+msgstr "Огдън - най-свестният мъж в града. Не мисля, че жена му ме харесва особено много, но докато тя продава бъчви, нямам нищо против това. Напоследък прекавам повече време с Огдън, отколкото с останалите, но поне той ме оценява...."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:419
 msgid "I wanna tell ya sumthin', 'cause I know all about this stuff. It's my specialty. This here is the best... theeeee best! That other ale ain't no good since those stupid dogs..."
-msgstr ""
+msgstr "Искам да ти споделя нещо, понеже разбирам от това. То ми е в кръвта. Това тук е най-доброто.... от най-най-доброто!  Другото пиво вече не струва откакто нехранимайковците...."
```

Comment:
```suggestion
msgstr "Искам да ти кажа нешщо, 'щото аз ги зная тия работи. То ми е в кръвта. Т'ва тук е най-доброто.... нааааааай-доброто!  Другото пиво вече не струва откакто  тия нехранимайковци...."
```

Reply:
Loving the use of slang here, makes things appear much more natural (since it's a drunkard talking)